app/controllers/invitations_controller.rb

```diff
@@ -1,8 +1,8 @@
 class InvitationsController < ApplicationController
-  before_filter :authorize
 
   # xhr only, tested in a request spec
   def create
+    authorise :invitation, :create?
```

Comment:
You are right, it's a typo it should be authorize.

Usually we check authorization based on a model but in this case it's based on a controller action that's why I'm using Headless policies https://github.com/elabs/pundit#headless-policies

I tested that Headless policies can infer the name of controller action too.